lib/classes/media/subdef.php

```diff
@@ -825,4 +803,42 @@ private function save()
 
         return $this;
     }
+
+    /**
+     * Return list of technical data and there attributes
+     *
+     * @return array
+     */
+    public static function getTechnicalFieldsList(){
+
+        $datas = [
```

Comment:
datas should better be declared (in the method or in the class) as "private static $technicalFieldsList..." so it is initialized only once in case of multiple calls to getTech...()
When it's not possible to initialize the static to a runtime-only value, initialize it with null, false, empty or whatever, and set the value only once with a if(isnull($x)).
Also : because 'name' is always set and unique, it can be used as the array key, and test like array_key_exists('name'... can be avoided.
btw the whole list could be in a silex/symfony container but that's another question, it's ok for now.